doc/source/computation.rst

```diff
@@ -695,18 +697,34 @@ all accept are:
   ``min_periods`` non-null data points have been seen.
 - ``center``: boolean, whether to set the labels at the center (default is False)
 
+.. _stats.moments.expanding.note:
 .. note::
 
    The output of the ``.rolling`` and ``.expanding`` methods do not return a
    ``NaN`` if there are at least ``min_periods`` non-null values in the current
-   window. This differs from ``cumsum``, ``cumprod``, ``cummax``, and
-   ``cummin``, which return ``NaN`` in the output wherever a ``NaN`` is
-   encountered in the input.
+   window. This differs from :meth:`~DataFrame.cumsum`,
+   :meth:`~DataFrame.cumprod`, :meth:`~DataFrame.cummax`,
+   and :meth:`~DataFrame.cummin`, which return ``NaN`` in the output wherever
+   a ``NaN`` is encountered in the input.
+
+   Please see the example below. In order to match the output of ``cumsum``
+   with ``expanding``, use :meth:`~DataFrame.fillna`.
+
+   .. ipython:: python
+
+        sn = pd.Series([1,2,np.nan,3,np.nan,4])
+
+        sn.expanding().sum()
```

Comment:
I would specify ``min_periods`` explicty here, as that is what we are highliting. (and maybe change the example a bit so ``min_periods=1`` and ``min_periods=2``) show different results.

you *may* need to make this a sub-section  rather than a note (as its getting long)